lib/matplotlib/backends/backend_cairo.py

```diff
@@ -92,39 +92,34 @@ def attr(field):
 
 
 class RendererCairo(RendererBase):
-    @cbook.deprecated("3.3")
-    @property
-    def fontweights(self):
-        return {
-            100:          cairo.FONT_WEIGHT_NORMAL,
-            200:          cairo.FONT_WEIGHT_NORMAL,
-            300:          cairo.FONT_WEIGHT_NORMAL,
-            400:          cairo.FONT_WEIGHT_NORMAL,
-            500:          cairo.FONT_WEIGHT_NORMAL,
-            600:          cairo.FONT_WEIGHT_BOLD,
-            700:          cairo.FONT_WEIGHT_BOLD,
-            800:          cairo.FONT_WEIGHT_BOLD,
-            900:          cairo.FONT_WEIGHT_BOLD,
-            'ultralight': cairo.FONT_WEIGHT_NORMAL,
-            'light':      cairo.FONT_WEIGHT_NORMAL,
-            'normal':     cairo.FONT_WEIGHT_NORMAL,
-            'medium':     cairo.FONT_WEIGHT_NORMAL,
-            'regular':    cairo.FONT_WEIGHT_NORMAL,
-            'semibold':   cairo.FONT_WEIGHT_BOLD,
-            'bold':       cairo.FONT_WEIGHT_BOLD,
-            'heavy':      cairo.FONT_WEIGHT_BOLD,
-            'ultrabold':  cairo.FONT_WEIGHT_BOLD,
-            'black':      cairo.FONT_WEIGHT_BOLD,
-        }
-
-    @cbook.deprecated("3.3")
-    @property
-    def fontangles(self):
-        return {
-            'italic':  cairo.FONT_SLANT_ITALIC,
-            'normal':  cairo.FONT_SLANT_NORMAL,
-            'oblique': cairo.FONT_SLANT_OBLIQUE,
-        }
+    fontweights = cbook.deprecated("3.3")(property(lambda self: {
+        100:          cairo.FONT_WEIGHT_NORMAL,
+        200:          cairo.FONT_WEIGHT_NORMAL,
+        300:          cairo.FONT_WEIGHT_NORMAL,
+        400:          cairo.FONT_WEIGHT_NORMAL,
+        500:          cairo.FONT_WEIGHT_NORMAL,
+        600:          cairo.FONT_WEIGHT_BOLD,
+        700:          cairo.FONT_WEIGHT_BOLD,
+        800:          cairo.FONT_WEIGHT_BOLD,
+        900:          cairo.FONT_WEIGHT_BOLD,
+        'ultralight': cairo.FONT_WEIGHT_NORMAL,
+        'light':      cairo.FONT_WEIGHT_NORMAL,
+        'normal':     cairo.FONT_WEIGHT_NORMAL,
+        'medium':     cairo.FONT_WEIGHT_NORMAL,
+        'regular':    cairo.FONT_WEIGHT_NORMAL,
+        'semibold':   cairo.FONT_WEIGHT_BOLD,
+        'bold':       cairo.FONT_WEIGHT_BOLD,
+        'heavy':      cairo.FONT_WEIGHT_BOLD,
+        'ultrabold':  cairo.FONT_WEIGHT_BOLD,
+        'black':      cairo.FONT_WEIGHT_BOLD,
+    }))
+    fontangles = cbook.deprecated("3.3")(property(lambda self: {
+        'italic':  cairo.FONT_SLANT_ITALIC,
+        'normal':  cairo.FONT_SLANT_NORMAL,
+        'oblique': cairo.FONT_SLANT_OBLIQUE,
+    }))
+    mathtext_parser = cbook.deprecated("3.4")(
+        property(lambda self: MathTextParser('Cairo')))
```

Comment:
This is the only one that doesn't accomplish your goal IMO of making it "easier" to see the deprecated function as a block. If instead you did

```suggestion
    fontweights = cbook.deprecated("3.3")(property(lambda self: _fontweights))
    fontangles = cbook.deprecated("3.3")(property(lambda self: _font_angles))
    mathtext_parser = cbook.deprecated("3.4")(
        property(lambda self: MathTextParser('Cairo')))
```

and added right above the class definition

```
# deprecated font property mappings for RendererCairo
_fontweights = {
        100:          cairo.FONT_WEIGHT_NORMAL,
        200:          cairo.FONT_WEIGHT_NORMAL,
        300:          cairo.FONT_WEIGHT_NORMAL,
        400:          cairo.FONT_WEIGHT_NORMAL,
        500:          cairo.FONT_WEIGHT_NORMAL,
        600:          cairo.FONT_WEIGHT_BOLD,
        700:          cairo.FONT_WEIGHT_BOLD,
        800:          cairo.FONT_WEIGHT_BOLD,
        900:          cairo.FONT_WEIGHT_BOLD,
        'ultralight': cairo.FONT_WEIGHT_NORMAL,
        'light':      cairo.FONT_WEIGHT_NORMAL,
        'normal':     cairo.FONT_WEIGHT_NORMAL,
        'medium':     cairo.FONT_WEIGHT_NORMAL,
        'regular':    cairo.FONT_WEIGHT_NORMAL,
        'semibold':   cairo.FONT_WEIGHT_BOLD,
        'bold':       cairo.FONT_WEIGHT_BOLD,
        'heavy':      cairo.FONT_WEIGHT_BOLD,
        'ultrabold':  cairo.FONT_WEIGHT_BOLD,
        'black':      cairo.FONT_WEIGHT_BOLD,
    }
_fontangles = {
        'italic':  cairo.FONT_SLANT_ITALIC,
        'normal':  cairo.FONT_SLANT_NORMAL,
        'oblique': cairo.FONT_SLANT_OBLIQUE,
    }
```

then I think you would accomplish your goal.